### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by apdavison**

### CODE REVIEW

File: doc/gettinghelp.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+============
+Getting help
+============
+
+In case of questions about **fairgraph**, please e-mail support@humanbrainproject.eu.
```
Comment (yzerlaut) [inline doc/gettinghelp.rst]: support@ebrains.eu ? or still support@humanbrainproject.eu

File: doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+   mgm = Organization("Metro-Goldwyn-Mayer")
+   mgm.save(client)
+   author = Person("Laurel", "Stan", "laurel@example.com", affiliation=mgm)
+   author.save(client)
+
+::
+
+   mgm.address = Address(locality='Hollywood', country='United States')
+   mgm.save(client)
```
Comment (yzerlaut) [inline doc/index.rst]: I think this code is broken. I get:
```
Traceback (most recent call last):
  File "test.py", line 11, in <module>
    mgm.save(client)
  File "/home/yzerlaut/anaconda3/lib/python3.7/site-packages/fairgraph/base.py", line 474, in save
    if self.id or self.exists(client):
  File "/home/yzerlaut/anaconda3/lib/python3.7/site-packages/fairgraph/base.py", line 396, in exists
    response = client.query_nexus(self.__class__.path, query_filter, context)
  File "/home/yzerlaut/anaconda3/lib/python3.7/site-packages/fairgraph/base.py", line 100, in path
    raise ValueError("namespace not set")
ValueError: namespace not set
```

Comment (yzerlaut) [reply doc/index.rst]: I made an update of the documentation in a previous pull request to solve (/hide) this:
https://github.com/HumanBrainProject/fairgraph/pull/18/commits/2e3eb801e2fbbca3efdf1e4c6b59c8f661a8ab27

File: doc/knowledgegraph.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+recorded using patch-clamp electrophysiology is represented by a Python object
+:class:`PatchedCell` whose attributes correspond to the metadata stored in that node
+_and_ to the semantic connections to other nodes.
+
+
+
+Alternatives
+------------
+
+.. todo:: write about KG Query API, pyxus, KG Query Python pip
```
Comment (yzerlaut) [inline doc/knowledgegraph.rst]: I thought you wanted to make the KG Query API the default, no ? Only after this release ?

Comment (apdavison) [reply doc/knowledgegraph.rst]: yes, after this release, because changing the default risks breaking existing code.

**Pull request merged**